recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "2020.01.24" %}
+{% set version = "2020.01.27" %}
```

Comment:
Am I supposed to upgrade this? Or should I bump the build number instead?

Reply:
Yes, this is fine